qa/L0_backend_python/python_test.py

```diff
@@ -61,6 +61,56 @@ def _infer_help(self, model_name, shape, data_type):
             output0 = result.as_numpy('OUTPUT0')
             self.assertTrue(np.all(input_data_0 == output0))
 
+    def _optional_input_infer(self, model_name, input0, input1):
+        with httpclient.InferenceServerClient("localhost:8000") as client:
+            shape = (1,)
+            if input0:
+                input0_numpy = np.random.randint(0,
+                                                 100,
+                                                 size=shape,
+                                                 dtype=np.int32)
+            else:
+                # Set the input0 dummy data for testing purposes.
```

Comment:
Not really "dummy" data, should say set to default input0 data that the model uses if not provided.